### DISCUSSION TITLE
Kimai installation forwards to "https://kimai.example.com/"

### DISCUSSION THREAD

**Discussion by klausjettkant** [kimai/kimai#4171] [On-Premise: Installation & Updates] [answered]
For staging purposes I duplicated the Kimai webroot and MySQL database and published it under a new Domain/ URL. On entering the webseite, Kimai forwards to "https://kimai.example.com/" and I can not access my duplicated environment. In Wordpress I would do some corrections to the Base-URL. Unfortunately I do not find anything regarding a change in domain name in the Kimai documentation or here in GitHub. Do you have any idea how I can fix this? Thank you a lot!!

**Comment by kevinpapst** [comment]
Kimai has nothing like that Wordpress setting.
The URL is calculated from the webserver environment.

This might help: https://symfony.com/doc/current/setup/web_server_configuration.html

**Reply by klausjettkant** [reply]
Thank you for answering so quickly! I have no access to webserver configuration due to a managed hosting provider. Kimai is running perfectly on a different URL / Webroot on the same server. But after duplicating files and database, the error from above occurs. I am just wondering, why Kimai rewrites https://my.domain.tld to https://kimai.examle.com. Any other place i can look in?

**Reply by klausjettkant** [reply, marked as answer]
We found a solution. After dupliucating the webspace and publishing under another domain, we fogot to activate a SSL-Certificate. Accessing the website with the https-prefix now leads to the correct content. Thank you for your help!
